runtime/gc_glue_java/ContractslotScanner.hpp

```diff
@@ -119,14 +119,14 @@ class MM_ContractSlotScanner : public MM_RootScanner
 		while (NULL != (region = regionIterator2.nextRegion())) {
 			if ((MEMORY_TYPE_NEW == (region->getTypeFlags() & MEMORY_TYPE_NEW))) {
 				MM_HeapRegionDescriptorStandardExtension *regionExtension = MM_ConfigurationDelegate::getHeapRegionDescriptorStandardExtension(env, region);
-				for (UDATA i = 0; i < regionExtension->_maxListIndex; i++) {
+				for (uintptr_t i = 0; i < regionExtension->_maxListIndex; i++) {
 					MM_UnfinalizedObjectList *list = &regionExtension->_unfinalizedObjectLists[i];
 					if (!list->wasEmpty()) {
 						J9Object *object = list->getPriorList();
 						while (NULL != object) {
 							J9Object *movePtr = object;
 							if ((movePtr >= (J9Object *)_srcBase) && (movePtr < (J9Object *)_srcTop)) {
-								movePtr = (J9Object *)((((UDATA)movePtr) - ((UDATA)_srcBase)) + ((UDATA)_dstBase));
+								movePtr = (J9Object *)((((uintptr_t)movePtr) - ((uintptr_t)_srcBase)) + ((uintptr_t)_dstBase));
```

Comment:
seems like excessive usage of '()' : could just be (uintptr_t)_dstBase instead of ((uintptr_t)_dstBase)
just a line above uses simple (J9Object *)_srcBase